### ISSUE TITLE
Open html output in new tab?

### ISSUE THREAD

**Issue by tsp-kucbd** [bugy/script-server#463]
We are very happy with the new html and plotly enabled output!
Sometimes our output is larger than the remaining script result canvas - is it possible to open a result page in a new tab or window? Alternatively being able to collapse or resize the Description and parameter section?

**Comment by bugy** [comment]
Hi @tsp-kucbd, I'll put it in the backlog.

But does the scroll work for you, in the output panel?

**Comment by tsp-kucbd** [comment]
Yes it does, but the output panel get quite small when using explanations in the description.


![example](https://user-images.githubusercontent.com/54399273/122392055-d6fc9c00-cf73-11eb-9e47-4061cccad20d.jpeg)

**Comment by bugy** [comment]
Ah, I see

Ye, I'm planning to make parameter/description panels collapsible
